src/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/LambdaCompiler.cs

```diff
@@ -26,6 +26,8 @@ internal interface ILocalCache
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Maintainability", "CA1506:AvoidExcessiveClassCoupling")]
     internal sealed partial class LambdaCompiler : ILocalCache
     {
+        private static int s_lambdaMethodIndex;
```

Comment:
>s_lambdaMethodIndex [](start = 27, length = 19)

can't find where this is incremented.

Reply:
Ah, you're right. I must have deleted that while doing a small refactor.